timesketch/lib/analyzers/login.py

```diff
@@ -0,0 +1,178 @@
+"""Sketch analyzer plugin for login and logoff events."""
+from __future__ import unicode_literals
+
+import logging
+
+from timesketch.lib import emojis
+from timesketch.lib.analyzers import interface
+from timesketch.lib.analyzers import manager
+
+
+LOGON_TYPES = {
+    u'0': u'Unknown',
+    u'2': u'Interactive',
+    u'3': u'Network',
+    u'4': u'Batch',
+    u'5': u'WindowsService',
+    u'7': u'UnlockScreen',
+    u'8': u'NetworkCleartext',
+    u'9': u'NewCredentials',
+    u'10': u'RemoteInteractive',
+    u'11': u'CachedInteractive'
+}
+
+
+def parse_evtx_logoff_event(event, string_list, emoji):
```

Comment:
Can we refactor this to only do the parsing/extraction in this function and return the "attributes" dict instead of updating the event? Feels like we are doing a little too much here. Same for the parse_evtx_logon_event() method below.

Reply:
done